runner/master/run.sh

```diff
@@ -165,7 +165,7 @@ echo
 echo ">>> startsection Checking networks <<<"
 echo "============================================================================"
 NETWORKNAME="moodle"
-NETWORK=$(docker network list -q --filter name="${NETWORKNAME}")
+NETWORK=$(docker network list -q --filter name="${NETWORKNAME}$")
```

Comment:
While I like it (really didn't know that PCRE is available there)... I was just guessing if, maybe,  we could make the ```NETWORKNAME``` parametrizable, still defaulting to "moodle", but allowing to create/use different ones. We already do that for other env variables. Sort of:

````
NETWORKNAME ="${NETWORKNAME:-moodle}"
````

Other than that, open to discussion. The patch looks perfect!